gulp/config.js

```diff
@@ -21,6 +21,7 @@ module.exports = () => {
     ],
     meta: ['src/meta/**/*'],
     definitions: ['src/**/definition.yml'],
+    icons: ['src/materials/icons/*.svg'],
```

Comment:
I have mixed feelings. On one hand, I understand you (icons tasks handling everything regarding icons). On the other hand, I think this tasks should only go the sprite generation (the rest is an asset). I could go either way, but if we go for first (icons tasks doing both copy AND sprite generation), this would make the assets glob more complex (I would basically need to blacklist the icons materials so they are not copied by the assets task).